src/text-field-components.tsx

```diff
@@ -106,8 +106,8 @@ export const HelperText = ({leftText, rightText, error, id}: HelperTextProps): J
                 </p>
             )}
             {rightText && (
-                <div className={classnames(styles.helperText)}>
-                    <Text1 color={rightColor} regular as="p" textAlign="right">
+                <div className={classnames(styles.helperText, {[styles.rightHelperText]: !!leftText})}>
+                    <Text1 color={rightColor} regular as="p" textAlign="right" wordBreak={false}>
```

Comment:
Before/after:

![input-fields-screenshot-test-tsx-text-field-multiline-and-max-length-with-long-helper-text-1-diff](https://github.com/user-attachments/assets/7cff3b21-4a37-430e-abb7-a06f930373ca)